### DISCUSSION TITLE
How can I get concurrent crew execution working

### DISCUSSION THREAD

**Discussion by FatherOfDragons** [crewAIInc/crewAI#1538] [Q&A] [answered]
I'm not quite sure whether this is a bug or a user error...  I cannot get multiple crews to execute concurrently.
For example 'write_a_book_with_flows' example dies silently without any error traces when multiple WriteBookChapterCrew crews are started in order to write chapters concurrently. 
Things work fine when I force serial execution, but as soon as there is any parallelism things break down.  I get the same broken behavior when I try to run crews concurrently in a ThreadPoolExecutor. 
Tried in the following environments:  
MacOS Sequoia. Python 3.11.9
Debian GNU/Linux 12. (running in Docker). Python 3.11.10
Scanned bug reports and discussions, but did not see current mentions of similar issues.  
Any help or advice would be greatly appreciated.

**Comment by ubaidxmughal** [comment]
Any Fix regarding this??

**Reply by greysonlalonde** [reply, marked as answer]
there's been work that surfaces in [docs](https://docs.crewai.com/en/learn/kickoff-async#native-async-execution-with-akickoff), please try that.
